android/build.gradle

```diff
@@ -30,6 +30,10 @@ if( newDbSize != null && newDbSize.isLong()) {
     dbSizeInMB = newDbSize.toLong()
 }
 
+def useAsyncTaskThreadPoolExecutor = rootProject.hasProperty('AsyncStorage_use_AsyncTaskThreadPoolExecutor') ? rootProject.properties['AsyncStorage_use_AsyncTaskThreadPoolExecutor'] : true
+
+
+
```

Comment:
Can we remove these superfluous newlines?

Reply:
@gaodeng Please just remove this extra line 🙏

Reply:
@Krizzu  line 32 or line 36 ?
![Screen Shot 2019-07-21 at 7 59 07 PM](https://user-images.githubusercontent.com/1118249/61590925-1688a500-abf2-11e9-8435-3449d41de12c.png)